sale_commission_settlement_report/readme/CONFIGURE.rst

```diff
@@ -0,0 +1,12 @@
+To show the partner in settlement lines:
+
+#. Go to *Settings > Sales > Quotations & Orders*:
+
+  * enable the setting "Show partner details in settlements report"
+
+
+To hide lines with amount equals to 0:
+
+#. Go to *Settings > Sales > Quotations & Orders*:
+
+  * enable the setting "Skip lines in settlements with zero amount"
```

Comment:
* enable the setting "Do not display lines with zero amount in settlements and its report"